common/bloodlines/wc_bloodlines.txt

```diff
@@ -382,11 +382,25 @@ ymiron = {
 	vrykul_opinion = 10
 	command_modifier = {
 		morale_offence = 0.05
-	}	
+	}
 	bloodline_effect_warrior_training = 1
 
 	inheritance = patrilineal
 	allow_bastards = yes
 	picture = GFX_bloodlines_placeholder_symbol_2
 	flags = { historical_bloodline bloodline_warrior_training }
-}
\ No newline at end of file
+}
+greymane = {
+	monthly_character_prestige = 0.15
+	
+	gilnean_opinion = 10
```

Comment:
The opinion is not declared in `Warcraft-Guardians-of-Azeroth\common\modifier_definitions\wc_opinion_modifier_definitions.txt`
See how other opinions are handled.